### PR TITLE
Gitlab feed token

### PR DESCRIPTION
**Pull request by babenek**
## Description

BASED ON: #354 

- Added Gitlab feed token pattern
- Updated benchmark skores with new markup https://github.com/Samsung/CredData/pull/31
- Freeze AnalysisTarget class to prevent 'line' modification and mistakes with line_len cached_property

## How has this been tested?

Please describe the tests that you ran to verify your changes.

- [ ] Test A
- [ ] Test B

### CODE REVIEW

File: .github/workflows/benchmark.yml [hunk 1]
```diff
@@ -62,7 +62,8 @@ jobs:
     - name: Checkout CredData
       uses: actions/checkout@v3
       with:
-        repository: Samsung/CredData
+        repository: babenek/CredData
+        ref: dev
```
Comment (csh519) [inline .github/workflows/benchmark.yml:66]:
```suggestion
        repository: Samsung/CredData
```

**Review by csh519 — COMMENTED** [review]
It seems https://github.com/Samsung/CredData/pull/31 should be merged first.
I'll review it.

**Comment by babenek** [comment]
@csh519 
#354 may be reviewed as base PR

File: credsweeper/file_handler/analysis_target.py [hunk 2]
```diff
@@ -3,7 +3,7 @@
 from typing import List, Optional
 
 
-@dataclass
+@dataclass(frozen=True)
 class AnalysisTarget:
```
Comment (csh519) [inline credsweeper/file_handler/analysis_target.py:7]: What about add `frozen` parameter to `DiffRowData` class of `utils.py` too?

**Review by csh519 — APPROVED** [review]
Thank you for adding Gitlab feed token.

LGTM :+1:

**Review by Yullia — APPROVED** [review]

**Pull request merged**